modin/pandas/base.py

```diff
@@ -4251,3 +4252,6 @@ def __array_ufunc__(
 
             return Series(pandas_result)
         return pandas_result
+
+    # namespace for additional Modin functions that are not available in Pandas
+    modin: ModinFunctions = CachedAccessor("modin", ModinFunctions)
```

Comment:
By implementing this method in this class, it also becomes available in the series (previously it was implemented only for the dataframe - erroneously).